docker-compose-debug.yml

```diff
@@ -9,13 +9,14 @@ services:
     image: tchiotludo/akhq
     ports:
      - "18080:8080"
+    env_file: .env
     environment:
       AKHQ_CONFIGURATION: |
         akhq:
           security:
             basic-auth:
-              - username: admin
-                password: d49745a74edfd20705ccaa277209db554b6b0d78c20780a4facf87e88aee78c0
+              - username: ${AKHQ_USER}
+                password: ${VAULT_PASS}
```

Comment:
Why have you named it VAULT_PASS ? akhq is not a vault.

Reply:
Ah! Sorry for that. That was suggested by copilot i guess.